software/goose-visual/src/interface/escape_codes.cpp

```diff
@@ -0,0 +1,6 @@
+#include "escape_codes.h"
+
+std::ostream & operator<<(std::ostream &stream, const EscapeCode &code) {
+	stream << static_cast<char>(27) << "[38;5;" << static_cast<int>(code) << "m";
```

Comment:
What is 27 magic number? Necessary to print colors?

Reply:
Escape ESC character, not printable